include/seastar/parquet/file_reader.hh

```diff
@@ -0,0 +1,475 @@
+#pragma once
+
+#include <seastar/parquet/exception.hh>
+#include <seastar/parquet/schema.hh>
+#include <seastar/parquet/parquet_types.h>
+#include <seastar/parquet/rle_encoding.hh>
+#include <seastar/parquet/overloaded.hh>
+
+#include <seastar/core/file.hh>
+#include <seastar/core/fstream.hh>
+#include <seastar/core/seastar.hh>
+
+#include <optional>
+#include <variant>
+
+namespace parquet {
+
+////////////////////////////////////////////////////////////////////////////////
```

Comment:
nitpick: these separating dashes will probably need to go, as they're not part of our codestyle

Reply:
Fixed in ac9cdae.